java.md

```diff
@@ -0,0 +1,99 @@
+# Java on Graviton
+
```

Comment:
need an intro paragraph about how most java is cross platform and runs well.

Maybe also something about corretto

Reply:
added in intro paragraphs to the java and python pages, added a blurb about corretto as well